app/uploaders/driver_id_uploader.rb

```diff
@@ -0,0 +1,47 @@
+class DriverIdUploader < CarrierWave::Uploader::Base
```

Comment:
Is there any reason to use Carrierwave for new project instead of ActiveStorage built in Rails?

Reply:
Andrea wanted me to use it